src/pkg/packager/sources/oci.go

```diff
@@ -29,15 +29,16 @@ var (
 
 // OCISource is a package source for OCI registries.
 type OCISource struct {
-	*types.ZarfPackageOptions
-	*zoci.Remote
+	Remote        *zoci.Remote
+	Src           string
```

Comment:
Was `OCISource.Source` unappealing? Seems different than what you were doing above.